fullstack/lib/LocalConnectionToCore.ts

```diff
@@ -0,0 +1,35 @@
+import Core from '@ulixee/hero-core';
+import { ConnectionToCore } from '@ulixee/hero';
+import ICoreRequestPayload from '@ulixee/hero-interfaces/ICoreRequestPayload';
+import ConnectionToClient from '@ulixee/hero-core/connections/ConnectionToClient';
+import TypeSerializer from '@ulixee/commons/TypeSerializer';
+
+export default class LocalConnectionToCore extends ConnectionToCore {
```

Comment:
If we change to what I proposed above, this would probably be ConnectionToLocalCore

Reply:
Agreed. Pushed your change.